tests/saitests/sai_qos_tests.py

```diff
@@ -1836,7 +1836,7 @@ def runTest(self):
             # Since there is variability in packet leakout in hwsku Arista-7050CX3-32S-D48C8 and 
             # Arista-7050CX3-32S-C32. Starting with zero pkts_num_leak_out and trying to find 
             # actual leakout by sending packets and reading actual leakout from HW
-            if hwsku == 'Arista-7050CX3-32S-D48C8' or hwsku == 'Arista-7050CX3-32S-C32':
+            if hwsku == 'Arista-7050CX3-32S-D48C8' or hwsku == 'Arista-7050CX3-32S-C32' and pkts_num_fill_min != 0:
```

Comment:
Why is the leakout number different for lossless vs lossy?

Reply:
@neethajohn leakout is not different for lossy and lossless. Here the problem is we don't have enough packets to leakout from the port in lossy case since fill_min is 0. So I like to avoid margin deduction in lossy case to have enough packets to send out of the port and fulfill leakout requirements.